### PR TITLE
Instruct how to get Optifine zoom in Zoomify

### PR DESCRIPTION
**Pull request by Imzxhir**
draft 1

### CODE REVIEW

**Comment by Imzxhir** [comment]
I’ve finished the instructions on how to get optifine zoom in zoomify.

**Comment by RaptaG** [comment]
> I’ve finished the instructions on how to get optifine zoom in zoomify.

I'm gonna make some suggestions

**Comment by Imzxhir** [comment]
> > I’ve finished the instructions on how to get optifine zoom in zoomify.
> 
> I'm gonna make some suggestions

Alright, go for it.

**Comment by RaptaG** [comment]
> Alright

It's all yours, a PR in a PR https://github.com/Imzxhir/wiki/pull/1

**Review by RaptaG — APPROVED** [review]

**Comment by RaptaG** [comment]
@Madis0, it's ready I believe.

File: en-us/give-up-optifine.md [hunk 1]
```diff
@@ -54,6 +54,15 @@ Here is the list of OptiFine features that are supported in this modpack:
 
 ² Snow is opt-in via settings as it changes some textures too, making it weird with some resource packs.
 
+³ The mod Zoomify is used to get the zoom function inside the modpack. The configuration of the mod provided by the modpack is not identical to the OptiFine zoom since most people prefer a smooth than an instant zoom. The following are instructions on how to customize the mod in order to make zooming identical to OptiFine:
```
Comment (Madis0) [inline en-us/give-up-optifine.md:57]: The ³ doesn't refer to anything

[the remark this answers is not in the document]
Comment (RaptaG) [reply en-us/give-up-optifine.md:57]: :thinking:

**Comment by Madis0** [comment]
I will not merge it just in case until the new Zoomify update is out, as that is said to have some more changes.

**Comment by RaptaG** [comment]
> I will not merge it just in case until the new Zoomify update is out, as that is said to have some more changes.

Yeah on the Scroll Zoom

File: en-us/give-up-optifine.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Madis0) [reply en-us/give-up-optifine.md:57]: Shorter, more concise

[the remark this answers is not in the document]
Comment (Imzxhir) [reply en-us/give-up-optifine.md:57]: Should I commit the suggestion?

[the remark this answers is not in the document]
Comment (RaptaG) [reply en-us/give-up-optifine.md:57]: Yeah

**Comment by Imzxhir** [comment]
Is this PR going to be merged because the upcoming update for Zoomify is going to be a scroll zoom overhaul which doesn't affect the instructions because all I say is to turn Enable Scroll Zoom off?

**Comment by Madis0** [comment]
We don't know what will be in Zoomify until it's out, plans can change. Besides, there is an unresolved comment above.

**Comment by Imzxhir** [comment]
> We don't know what will really be in Zoomify until it's out, plans can change. Besides, there is an unresolved comment above.

You can check the changelog file in the zoomify repo to see what's going to be in the next update.

File: en-us/give-up-optifine.md (hunk 1 above)
Comment (Madis0) [reply en-us/give-up-optifine.md]: It still doesn't refer to anything. See how I've done it with the other superscript numbers.

Comment (Imzxhir) [reply en-us/give-up-optifine.md]: If you look on my fork / patch-1, you can see that I don't have the same footer as the one before. I merged the pull request that you did and now it's different to the one before.

Comment (Madis0) [reply en-us/give-up-optifine.md]: That's true, but I'm not referring to that line. I mean that line 27 lacks the `³`, I just cannot comment on lines you haven't edited.

Though now that I look at it, it should even be `²` due to the order and `³` should be the better snow message.

Comment (Imzxhir) [reply en-us/give-up-optifine.md]: I've added ³ to line 27, anything else hat needs to be chnaged / done?

Comment (Madis0) [reply en-us/give-up-optifine.md]: Make it a ² because of the order and make better snow the ³.

Comment (Imzxhir) [reply en-us/give-up-optifine.md]: Done!

**Comment by Imzxhir** [comment]
Why is my commit taking so long to merge?

**Comment by Madis0** [comment]
> Why is my commit taking so long to merge?

I already told you that, twice. Please be respectful and patient.

And I was right too - testing current Zoomify 1.7.1 preview out made me notice an oversight in those instructions.

File: en-us/give-up-optifine.md [hunk 2]
```diff
@@ -50,9 +50,26 @@ Here is the list of OptiFine features that are supported in this modpack:
...
-² Snow is opt-in via settings as it changes some textures too, making it weird with some resource packs.
+1. [Disable this mod](disabling-mods.md)
+2. [Install](adding-more-mods.md) the original [Cull Leaves](https://www.curseforge.com/minecraft/mc-mods/cull-leaves)
+3. Enable Cull Leaves' bundled resource pack
+4. Disable "Use Block Face Culling" in <kbd>Options</kbd> -> <kbd>Video Settings...</kbd> -> <kbd>Advanced</kbd>.
+
+² The defaults do not match OptiFine, because people preferred a smoother zoom. To make it identical to OptiFine:
+
+1. Click <kbd>Mods</kbd> button, search for `Zoomify`, click it and on the top left hit the `Configure` icon.
+2. In the Behaviour tab, set Initial Zoom to `4` and Zoom Transition to `Instant`
```
Comment (Madis0) [inline en-us/give-up-optifine.md:63]:
```suggestion
2. In the Behaviour tab, set Zoom Transition to `Instant`
```
Initial Zoom already is 4 as of Zoomify 1.7.1.

**Comment by Imzxhir** [comment]
> > Why is my commit taking so long to merge?
> 
> I already told you that, twice. Please be respectful and patient.
> 
> And I was right too - testing current Zoomify 1.7.1 preview out made me notice an oversight in those instructions.

Yeah. That's my bad, sorry

**Comment by Imzxhir** [comment]
> > Why is my commit taking so long to merge?
> 
> I already told you that, twice. Please be respectful and patient.
> 
> And I was right too - testing current Zoomify 1.7.1 preview out made me notice an oversight in those instructions.

Do you think you can send me a 1.7.1 preview version of zoomify? I want to test out some things with the settings and instructions

**Comment by Madis0** [comment]
> Do you think you can send me a 1.7.1 preview version of zoomify? I want to test out some things with the settings and instructions

It's right here: https://github.com/isXander/Zoomify/actions
1. Click whatever commit is the latest at the time
2. Download "artifact"
3. Unpack the jar from inside the zip to your mods folder
4. Considering that I plan to remove the FO-specific config anyway when this update happens, you can delete the zoomify configs from `config` and `config/yosbr/config` before launching the game

**Comment by Imzxhir** [comment]
I have done some testing with the 1.7.1 version of Zoomify with the optifine preset, and it still works. No settings needed to be changed and the settings are identical to optifine zoom.

**Pull request merged**